src/pages/City/CityDetailDrawer/CityDetailDrawer.tsx

```diff
@@ -143,38 +122,17 @@ const CityDetailDrawer = (props: Props) => {
         <Row justify="center" gutter={[12, 0]}>
           <Col md={12} xs={24}>
             <Form.Item
-              name="officeNumber"
-              label="Номер офісу/квартири"
-              labelCol={{ span: 24 }}
-              initialValue={
-                props.city.officeNumber?.length > 0
-                  ? props.city.officeNumber
-                  : "---"
-              }
-            >
-              <Input
-                value={
-                  props.city.officeNumber?.length > 0
-                    ? props.city.officeNumber
-                    : "---"
-                }
-                disabled
-              />
-            </Form.Item>
-          </Col>
-          <Col md={12} xs={24}>
-            <Form.Item
-              name="postIndex"
-              label="Поштовий індекс"
+              name="level"
+              label="Рівень"
               labelCol={{ span: 24 }}
               initialValue={
-                props.city.postIndex?.length > 0 ? props.city.postIndex : "---"
+                props.city.level > 0 ? props.city.level : "---"
               }
             >
               <Input
                 value={
-                  props.city.postIndex?.length > 0
```

Comment:
Don't use both `initialValue` attribute in `Form.Item` and `value` attribute in `Input`. 
Use only one, preferably `initialValue` attribute in `Form.Item`.